src/components/PlaySongs/PlaySongs.tsx

```diff
@@ -16,24 +16,26 @@ import { Container,
     ButtonCanBeDisabled,
     YoutubeWrapper, 
     Button, 
-    Option, 
     Close,
-    OptionContainer,
+    PartyWrapper,
  } from "./styles";
 import Song from "../../types/Song";
 import { playlistsAsyncActions } from "../../store/slices/playlists/slice";
 import { selectCurrentUser } from "../../store/slices/authentication/selectors";
 import Playlist from "../../types/Playlist";
 import User from "../../types/User";
 
-const PlaySongs = () => {
+type playSongsProps = {
+    isParty: boolean,
+    closePlayer: () => void,
+}
+
+const PlaySongs = (props: playSongsProps) => {
+    const {isParty} = props
```

Comment:
instead of this:
{ isParty, closePlayer }: playSongsProps

Reply:
fixed